db/migrate/20201103190022_add_image_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageToItems < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default(:items, :image, 'https://breakthrough.org/wp-content/uploads/2018/10/default-placeholder-image.png')
+  end
```

Comment:
The database default image did not work as expected when updated with an empty image string `""`. Had to create a set_default_image method, but perhaps there are better solutions?